build.sh

```diff
@@ -151,6 +152,11 @@ fi
 if hasArg --codecov; then
     CUML_EXTRA_PYTHON_ARGS="${CUML_EXTRA_PYTHON_ARGS} --linetrace=1 --profile"
 fi
+if hasArg --ccache; then
+    CUML_EXTRA_CMAKE_ARGS="${CUML_EXTRA_CMAKE_ARGS} -DCMAKE_C_COMPILER_LAUNCHER=ccache"
+    CUML_EXTRA_CMAKE_ARGS="${CUML_EXTRA_CMAKE_ARGS} -DCMAKE_CXX_COMPILER_LAUNCHER=ccache"
+    CUML_EXTRA_CMAKE_ARGS="${CUML_EXTRA_CMAKE_ARGS} -DCMAKE_CUDA_COMPILER_LAUNCHER=ccache"
```

Comment:
I think creating a single `USE_CCACHE` option that turns all of this in the `CMakeLists` would be a great addition, and would make it easier for people that invoke cmake directly, what do you think?

Reply:
Great idea! Just added it along with your other suggestion.